patches/minecraft/net/minecraft/advancements/AdvancementRewards.java.patch

```diff
@@ -1,5 +1,28 @@
 --- a/net/minecraft/advancements/AdvancementRewards.java
 +++ b/net/minecraft/advancements/AdvancementRewards.java
+@@ -29,12 +_,22 @@
+    private final ResourceLocation[] f_9980_;
+    private final ResourceLocation[] f_9981_;
+    private final CommandFunction.CacheableFunction f_9982_;
++   private final net.minecraftforge.common.advancements.ICustomAdvancementReward[] customRewards;
++
++   public AdvancementRewards(int experience, ResourceLocation[] loot, ResourceLocation[] recipes, CommandFunction.CacheableFunction function, net.minecraftforge.common.advancements.ICustomAdvancementReward[] customRewards) {
++      this.f_9979_ = experience;
++      this.f_9980_ = loot;
++      this.f_9981_ = recipes;
++      this.f_9982_ = function;
++      this.customRewards = customRewards;
++   }
+ 
+    public AdvancementRewards(int p_9985_, ResourceLocation[] p_9986_, ResourceLocation[] p_9987_, CommandFunction.CacheableFunction p_9988_) {
```

Comment:
The old constructor should probably be deprecated

Reply:
Idk as it still works just the same as passing an empty array for the custom rewards